topsort-api-v2.yml

```diff
@@ -262,8 +262,14 @@ components:
           format: int32
           minimum: 1
           description: Specifies the maximum number of auction winners that should be returned.
-        # category:
-        #   $ref: '#/components/schemas/Category'
+        # categories:
+        #   type: array
+        #   description: >
+        #     An array containing the categories of the bids that will participate in an auction.
+        #     The product associated to the bid must belong to all of the categories in the request.
```

Comment:
This is not necessarily true, in case they use the campaign api and create a campaign for a product with a different category. Ie, there could be a bid for an ipad in the android tablets category. We don't validate that in any part of the code (campaign, nor auction engine).

Reply:
You're right. The product must belong to the category or the bid should have set the category as the trigger through the campaign API; both are valid use cases.

Reply:
After talking with @fcifuentesq we've decided not to allow using categories as the trigger for sponsored listings in the campaigns API. Therefore the description stands.